ozaria/site/components/interactive/PageInteractive/draggableOrdering/index.vue

```diff
@@ -0,0 +1,127 @@
+<script>
+  import BaseInteractiveTitle from '../BaseInteractiveTitle'
+  import OrderingSlots from './OrderingSlots'
+
+  export default {
+    components: {
+      'base-interactive-title': BaseInteractiveTitle,
+      'ordering-slots': OrderingSlots
+    },
+
+    props: {
+      interactive: {
+        type: Object,
+        required: true
+      },
+
+      introLevelId: {
+        type: String,
+        required: true
+      },
+
+      interactiveSession: {
+        type: Object,
+        default: undefined
+      },
+
+      courseInstanceId: {
+        type: String,
+        default: undefined
+      }
+    },
+
+    data () {
+      return {
+        draggableGroup: Math.random().toString(),
+
+        promptSlots: [
+          { id: 1, text: 'one option' },
+          { id: 2, text: 'two option' },
+          { id: 3, text: 'three option' },
+          { id: 4, text: 'four option' }
+        ],
+
+        answerSlots: []
+      }
+    }
+  }
+</script>
+
+<template>
+  <div class="insert-code-container">
+    <base-interactive-title
+      :interactive="interactive"
+    />
+
+    <div class="prompt-row">
+      <ordering-slots
```

Comment:
Do we really need ordering-slots component? It seems we are using that component to only set slotData, and then send it to BaseDraggableSlot. Does it make sense to compute `slotData` here itself and use BaseDraggableSlot directly? Is there any other purpose of ordering-slots component?

Reply:
I think it's a nice interface to have. But what you say is true. I am in favor of keeping it currently. When working on the UI pieces it will be good to keep this in mind.